libsvmdata/datasets.py

```diff
@@ -45,6 +42,23 @@
 }
 
 
+# LIBSVMDATA_PATH is determine using environment variables.
+# The top priority is LIBSVM_DATA_HOME which is specific to this package.
+# If XDG_DATA_HOME is set, the data will be put in a subfolder 'libsvm'.
+# Finally, the default value is set to $HOME/libsvm_data.
+def get_data_home():
+    data_home = os.environ.get(
+        'LIBSVM_DATA_HOME', os.environ.get('XDG_DATA_HOME', None)
```

Comment:
I don't have this variable set in any of my systems. But it seems standard.

LGTM

Reply:
LGTM too